Kopernicus/Kopernicus/RuntimeUtility/RnDFixer.cs

```diff
@@ -40,14 +40,14 @@ namespace Kopernicus
     /// </summary>
     public class RnDFixer : MonoBehaviour
     {
-        List<RDPlanetListItemContainer> stars = new List<RDPlanetListItemContainer>();
+        internal static List<RDPlanetListItemContainer> RDRotationKill = new List<RDPlanetListItemContainer>();
```

Comment:
`RnDRotationKill`?

Reply:
it's the list of bodies for which we want to "kill" the rotation

it's not required that they are stars, but someone might want to kill the rotation of other bodies

I initially had named it stars  because I didn't intend to allow the users to choose which bodies to select

Reply:
I know. I was just suggesting a new name (with an n)